### DISCUSSION TITLE
Module not found error

### DISCUSSION THREAD

**Discussion by Megatronus213** [Discord-OTP-Forcer/Discord-OTP-Forcer#138] [Community Support]
![Screenshot (398)](https://github.com/user-attachments/assets/f3b8c4b5-91ae-4c29-a971-24431b666903)
Also i have the python (3.10)and google version(updated), nothing else  
also my dis. account got hacked and i need to change password to access it, but same 2FA issue

**Comment by Megatronus213** [comment]
how do i download that, i dont see a download button

**Comment by Megatronus213** [comment]
for future references , can u make a yt tutorial , also i sent a friend
request at discord alpha_onslaught

**Reply by FarhanMLG** [reply]
open cmd, then "pip install webdriver_manager"
